tests/Kernel/Paragraphs/MediaParagraphsTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel\Paragraphs;
+
+use Drupal\media\Entity\Media;
+use Drupal\paragraphs\Entity\Paragraph;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the rendering of paragraph types with media fields.
+ */
+class MediaParagraphsTest extends ParagraphsTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'language',
+    'content_translation',
+    'paragraphs',
+    'user',
+    'system',
+    'file',
+    'field',
+    'entity_reference_revisions',
+    'datetime',
+    'image',
+    'link',
+    'text',
+    'filter',
+    'options',
+    'oe_paragraphs',
+    'media',
+    'oe_media',
+    'oe_paragraphs_media',
+    'allowed_formats',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installEntitySchema('media');
+    $this->installConfig([
+      'media',
+      'oe_media',
+      'oe_paragraphs_media',
+    ]);
+  }
+
+  /**
+   * Test text with featured media paragraph rendering.
+   */
+  public function testTextWithMedia(): void {
```

Comment:
Do we have any assertions that the paragraph still renders correctly with missing media info (early returns)?

Reply:
Extra steps added.